### PR TITLE
Translation

### PR DESCRIPTION
**Pull request by ghost**
Missing translations.
Some corrections.

### CODE REVIEW

File: translations/de.ts [hunk 1]
```diff
@@ -89,62 +94,120 @@
...
     </message>
     <message>
-        <location filename="../src/widget/form/addfriendform.cpp" line="116"/>
-        <source>This address does not exist</source>
-        <comment>The DNS gives the Tox ID associated to toxme.se addresses</comment>
-        <translation type="unfinished"></translation>
+        <location filename="../src/widget/form/addfriendform.cpp" line="100"/>
+        <source>qTox needs to use the Tox DNS, but can&apos;t do it through a proxy.
+Ignore the proxy and connect to the Internet directly?</source>
+        <translation>qTox muss die Tox DNS nutzen, dies klappt allerdings nicht über einen Proxy.
```
Comment (cebe) [inline translations/de.ts]: Besser ohne Artikel: "qTox muss Tox DNS benutzten, ..."
Wenn mit Artikel sollte es "das DNS" sein, nicht "die DNS". DNS = Dynamic naming system, also "das System"

See also vanished translation below.

Comment (ghost) [inline translations/de.ts]: Am 10.12.2014 um 09:22 schrieb Carsten Brandt:

> In translations/de.ts
> https://github.com/tux3/qTox/pull/917#discussion-diff-21589135:
> 
> > ```
> >  </message>
> >  <message>
> > ```
> > -        <location filename="../src/widget/form/addfriendform.cpp" line="116"/>
> > -        <source>This address does not exist</source>
> > -        <comment>The DNS gives the Tox ID associated to toxme.se addresses</comment>
> > -        <translation type="unfinished"></translation>
> > -        <location filename="../src/widget/form/addfriendform.cpp" line="100"/>
> > -        <source>qTox needs to use the Tox DNS, but can&apos;t do it through a proxy.
> >   +Ignore the proxy and connect to the Internet directly?</source>
> > -        <translation>qTox muss die Tox DNS nutzen, dies klappt allerdings nicht über einen Proxy.
> 
> Besser ohne Artikel: "qTox muss Tox DNS benutzten, ..."
> Wenn mit Artikel sollte es "das DNS" sein, nicht "die DNS". DNS =
> Dynamic naming system, also "das System"

IMO mit Artikel besser -> "das"

## 

PGP-Schlüssel:
0xD2A804E4

**Comment by zetok** [comment]
Could you please squash those commits?

File: translations/de.ts (hunk 1 above)
Comment (cebe) [inline translations/de.ts]: fine with me.

**Comment by dubslow** [comment]
Run

```
git checkout -b detr
git checkout master
git reset --hard ce6e50b912
git pull https://github.com/tux3/qTox.git master;
git checkout detr
git rebase -i master
```

and be sure to mark all but the first commit as "squash" instead of "pick".

**Comment by tux3** [comment]
It seems that most of the missing translations were added in the meantime.
I tried to rebase this PR on master, but the changes are way out of date now (also, I don't speak german).

Please try to rebase it on master or start another branch if you feel some corrections are still needed, but I'm closing this for now.

**Pull request closed without merging**